src/ts/src/common/api/index.tsx

```diff
@@ -2,6 +2,10 @@
 import axios from "axios";
 import { jsonToType } from "src/common/utils";
 
+export const DEFAULT_FETCH_OPTIONS: RequestInit = {
```

Comment:
shifting to use browser's native Fetch API instead of axios

Reply:
why are we shifting?

Reply:
Great question! Axios adds extra weight to the browser unnecessarily when the native Fetch API works just fine 👌